### PR TITLE
Create cheuk.dev.json

### PR DESCRIPTION
**Pull request by Cheukting**
Adding Cheuk's blog at https://cheuk.dev

### CODE REVIEW

File: blogs/cheuk.dev.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "title": "Cheuk Ting Ho", //required
```
Comment (cosimameyer) [inline blogs/cheuk.dev.json:2]: Can you remove the `\\optonal` and `\\required` comments? I believe it caused problems when parsing the files in the past. Thanks! ☺️

File: blogs/cheuk.dev.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "authors": [ //required
+    {
+      "name": "Cheuk Ting Ho", //required
+      "social_media": [{
+         "twitter": "@cheukting_ho",
+         "mastodon": "@cheukting_ho@fosstodon.orgg",
+         "github": "Cheukting",
+         "youtube": "%40cheuktingho",
+         "website": "https://cheuk.dev",
+         "linkedin": "cheukting-ho",
```
Comment (cosimameyer) [inline blogs/cheuk.dev.json:19]: And also the trailing comma here? Thank you! ☺️

**Review by cosimameyer — CHANGES_REQUESTED** [review]
Thanks so much for contributing 🥳 It looks great! There are just two small changes needed and then I can merge the PR.

**Comment by Cheukting** [comment]
Updated, please check :-)

**Review by cosimameyer — APPROVED** [review]
Thanks, it looks great! 🤩

**Pull request merged**